dipy/viz/colormap.py

```diff
@@ -255,3 +261,52 @@ def line_colors(streamlines, cmap='rgb_standard'):
                     for streamline in streamlines]
 
     return np.vstack(col_list)
+
+
+lowercase_cm_name = {'blues': 'Blues', 'accent': 'Accent'}
+
+
+def create_colormap(v, name='jet', auto=True):
```

Comment:
"jet" is a really bad colormap (see e.g., ). Can we use switch to using a different default (viridis??)? If we want to keep this as a default to support back-compatibility, can we at least add a warning that we will change this in the future?